frontend/src/app/testResults/uploads/Uploads.tsx

```diff
@@ -135,10 +136,10 @@ const Uploads = () => {
   };
 
   return (
-    <div className="grid-row">
+    <div className="grid-row header-size-fix">
       <div className="prime-container card-container">
         <div className="usa-card__header">
-          <h2>Upload your results</h2>
+          <h1>Upload your results</h1>
```

Comment:
I see our `Success: File Accepted` and `Error: File not accepted` messages are using `<h4>` tags
I don't think these should be headings at all tbh, but at least we should not be skipping heading levels

Reply:
fixed